lib/code_climate/test_reporter/calculate_blob.rb

```diff
@@ -1,16 +1,15 @@
 module CodeClimate
   module TestReporter
     class CalculateBlob
-
       def initialize(file_path)
         @file_path = file_path
       end
 
       def blob_id
-        calculate_with_file or calculate_with_git
+        calculate_with_file || calculate_with_git
```

Comment:
ah, one last note: we recently updated our styleguide to allow this kind of usage of `or` (https://github.com/codeclimate/styleguide/commit/3aa6d71925acad7b877f7afd5a6aee971f9df332) so might be worth considering leaving this one, but up to you